src/main/java/uk/gov/companieshouse/controller/DissolutionDirectorController.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.companieshouse.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.exception.BadRequestException;
+import uk.gov.companieshouse.exception.DissolutionNotFoundException;
+import uk.gov.companieshouse.exception.NotFoundException;
+import uk.gov.companieshouse.model.dto.dissolution.DissolutionDirectorPatchRequest;
+import uk.gov.companieshouse.model.dto.dissolution.DissolutionDirectorPatchResponse;
+import uk.gov.companieshouse.service.dissolution.director.DissolutionDirectorService;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+import static uk.gov.companieshouse.util.EricHelper.getEmail;
+
+@RestController
+@RequestMapping("/dissolution-request/{company-number}/directors/{director-id}")
+public class DissolutionDirectorController {
+    private final DissolutionDirectorService dissolutionDirectorService;
+
+    public DissolutionDirectorController(DissolutionDirectorService dissolutionDirectorService) {
+        super();
```

Comment:
suoer? Why?

Reply:
Good question. I copypasted it from DissolutionController. Any clue why DissolutionController has it?

Reply:
no idea - try removing it